src/filters/time/time.constants.ts

```diff
@@ -8,8 +8,46 @@ const TIME = {
     FILLER_WORDS,
 };
 
+const RELATIVE_PREPOSITIONS = '(in|after) ';
+const RELATIVE_POSTPOSTIONS = ' (from now|later|after)';
+const HOURS = '(hours|hour|hrs|hr)';
+const MINUTES = '(minutes|minute|mins|min)';
+const SECONDS = '(seconds|second|secs|sec)';
+const TIME_UNITS = {
+    ANY: `(${HOURS}|${MINUTES}|${SECONDS})`,
+    HOURS: `${HOURS}`,
+    MINUTES: `${MINUTES}`,
+    SECONDS: `${SECONDS}`,
+};
+
+const HALF = '(half an|half a)';
+const QUARTER = '(a quarter of a|a quarter of an)';
+const ONE = '(a|an|one)';
+
+const VERBAL_QUANTIFIERS = {
+    ANY: `(${HALF}|${QUARTER}|${ONE})`,
+    HALF: `${HALF}`,
+    QUARTER: `${QUARTER}`,
+    ONE: `${ONE}`,
+};
+
+const ARGUMENT = `([0-9]+|${VERBAL_QUANTIFIERS.ANY}) ${TIME_UNITS.ANY}`;
+const ARGUMENT_AFTER = `${RELATIVE_PREPOSITIONS}${ARGUMENT}`;
+const ARGUMENT_FIRST = `${ARGUMENT}${RELATIVE_POSTPOSTIONS}`;
+
+const RELATIVE_TIME = {
```

Comment:
Can we also move this to a different folder as it is relative time logic. And go back to return TIME as default export?